zt_frontend/src/components/ComponentWrapper.vue

```diff
@@ -115,6 +128,23 @@ export default {
           ),
       };
     },
+    getEventBindings(component: any) {
```

Comment:
Reason for this method?

Reply:
@HarshitaDPoojary @metc1999

Reply:
Working even after removing the method.
Removed in [today's commit](https://github.com/Zero-True/zero-true/pull/277/commits/644eb127e48c34edf17c1178188def6aa83e650d#diff-ddd1ef94f7d87d0784706beb6ce11cf9aaf773f460a0852cfdc80ddb9ff82c22L131)